scripts/update_all_products.pl

```diff
@@ -217,6 +245,22 @@
 			}
 		}
 
+		# Fix products that were created without the lc field, but that have a lang field
+		if ($fix_missing_lc) {
```

Comment:
Wouldn't it be cleaner to use `if ($fix_missing_lc and (not defined $product_ref->{lc})) {` here and simplify the following `if` statements? There's probably not a huge advantage in printing `$product_ref->{lang}` for every product?

Reply:
Right, I will change it, thanks!